src/main/resources/schema.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS student (
```

Comment:
Ideally, IF NOT EXISTS is used for standalone apps. If you have a running web app, it's not preferred to add the clause.

If you do so and happen to remove a table for some reason, it will recreate the table for no good reason.